### ISSUE TITLE
ComboBox with placeholder does not have valid aria-attr-value

### ISSUE THREAD

**Issue by kejatura** [microsoft/fluentui#12556]
### Environment Information

- **Package version(s)**: 7.98.0
- **Browser and OS versions**: Chrome, Edge

### Describe the issue:

When using ComboBox with placeholder prop, there is an accessibility issue when using App Insights - Fast Pass. See below:

![image](https://user-images.githubusercontent.com/58444231/78587496-e0c43480-77f1-11ea-8654-1cf5859fd18e.png)

### Please provide a reproduction of the issue in a codepen:

Code pen: https://codepen.io/kejatura/pen/YzXmWXa?editors=1010
1. Enable drop down in combobox, and leave drop down list as is.
2. Run App Insights - Fast Pass, and notice error in input field.

#### Actual behavior:

There is an invalid aria attribute: aria-activedescendant, where combobox has a '-listundefined' in its value.  

#### Expected behavior:

'-listundefined' should contain the number of items in list, i.e: '-list4', besides undefined; to ensure this passes the accessibility fast pass test.

**Comment by joschect** [comment]
Hi @kejatura this is definitely a bug! We'll get it fixed. Thanks for providing the codepen and all the details!
